project_event/models/project_project.py

```diff
@@ -54,6 +54,11 @@ class Project(models.Model):
         default='draft',
         track_visibility='onchange',
     )
+    sector_id = fields.Many2one(
```

Comment:
better to move this field under the client_type field.
this behaviour is missing: When we change the partner_id the Client_type value must be the value of the client_type of the tag (category_id) of the client

Reply:
@rimbendhaou : Changed